Lib/asyncio/taskgroups.py

```diff
@@ -198,7 +201,8 @@ def _on_task_done(self, task):
             return
 
         self._errors.append(exc)
-        if self._is_base_error(exc) and self._base_error is None:
+        is_base_error = self._is_base_error(exc)
```

Comment:
Hi @DontPanicO, I'm trying to understand why `is_base_error` is needed. Please help me understand why that is preferable to `self._is_base_error(exc)`. Thanks!

Reply:
Hi @willingc . I stored `self._is_base_error(exc)` result in a variable since it's used twice: once on line 205 and once on line 238